cmd/cmdtrace/cmd_span.go

```diff
@@ -53,6 +54,9 @@ func Setup(cmd *cobra.Command, dockerCli command.Cli) error {
 		ctx,
 		"cli/"+strings.Join(commandName(cmd), "-"),
 	)
+	cmdSpan.SetAttributes(attribute.StringSlice("cliArgs", args))
+	cmdSpan.SetAttributes(attribute.StringSlice("cliFlags", getFlags(cmd.Flags())))
```

Comment:
Since span attributes are a single-level map, so far I've been separating logical groups with `.`, e.g. `cli.args` / `cli.flags`.

(I'm realizing I should add a `TRACING.md` to the repo here with some info...serious 🚌 factor right now)